### ISSUE TITLE
Game Crashes when selecting a resolution higher than that of the monitor in fullscreen mode.

### ISSUE THREAD

**Issue by PanosPetras** [PanosPetras/The_Great_War#37]
We have to find a way to prevent the player from selecting a resolution higher than that of the monitor.

**Comment by TedLyngmo** [comment]
I just made two commits that should make it easier to figure out where it goes awry.

**Comment by PanosPetras** [comment]
> I just made two commits that should make it easier to figure out where it goes awry.

I actually think that it makes sense that the behavior of the app would be at least unexpected if we try to run in in fullscreen in a resolution higher that the monitor's native resolution. I think the most logical fix to this problem is to get the monitor's native resolution and use it as a limit. That's actually what most games do(I think).

**Comment by TedLyngmo** [comment]
Yes, that sounds like a good idea. I assume that even lower resolutions may fail too though.

**Comment by PanosPetras** [comment]
> I assume that even lower resolutions may fail too though.

No this is not the case. I tried it. It works fine, and it actually makes sense. If you run the game in fullscreen in a lower resolution, you are just downsizing your monitor.

**Comment by PanosPetras** [comment]
This issue is no longer reproducible
